search-sentiments-map/routes/update-data.js

```diff
@@ -21,7 +21,8 @@ const search = require('./search.js');
 // Scheduled to run at minute 0 past every 12th hour (11am and 11pm every day).
 router.get('/', (req, res) => {
   console.log('Updating Search Results Data.');
-  // trends.updateTrendsFunction(); Undo comment during final deploy.
+  // TODO(all): Uncomment out for testing or for final deploy
```

Comment:
list out your ldaps instead of writing 'all'. The reason we use ldaps is so that doing a search will highlight what action items you have.